generators/server/templates/_travis.yml

```diff
@@ -46,6 +52,18 @@ script:
 <%_ } else if (buildTool === 'gradle') { _%>
   - ./gradlew bootRepackage -Pprod -x test
 <%_ } _%>
+  - docker-compose up -d
+  - sleep 30s
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+<%_ if (buildTool === 'maven') { _%>
+  - ./mvnw spring-boot:run &
```

Comment:
not sure it will work if the configuration is uaa, so maybe exlcude this config ?